examples/getting_started/run.py

```diff
@@ -72,11 +169,11 @@
     if args.benchmark:
         end_time_seconds = time.time()
         elapsed_time_seconds = end_time_seconds - start_time_seconds
-        spear.log("Average frame time: %0.4f ms (%0.4f fps)" % ((elapsed_time_seconds / NUM_STEPS)*1000.0, NUM_STEPS / elapsed_time_seconds))
+        spear.log("    Average frame time: %0.4f ms (%0.4f fps)" % ((elapsed_time_seconds / NUM_STEPS)*1000.0, NUM_STEPS / elapsed_time_seconds))
```

Comment:
No need to indent this because it's not underneath a header. The purpose of indenting should be to logically group the indented statements underneath some kind of organizing header.

Reply:
Ah, that makes sense. Updated this.